static/js/message_edit.js

```diff
@@ -976,13 +976,14 @@ export function delete_topic(stream_id, topic_name, failures = 0) {
         data: {
             topic_name,
         },
-        success() {},
-        error(xhr) {
-            if (failures >= 9) {
-                // Don't keep retrying indefinitely to avoid DoSing the server.
-                return;
-            }
-            if (xhr.status === 502) {
+        success(data) {
+            if (data.result === "partially_completed") {
+                if (failures >= 9) {
+                    // Don't keep retrying indefinitely to avoid DoSing the server.
+                    return;
+                }
+
+                failures += 1;
```

Comment:
This line looks duplicated on line 995, below

Reply:
Ouch, thanks for catching that